testfiles/CMakeLists.txt

```diff
@@ -328,6 +328,8 @@ ADD_SIMULATION_TEST(IDF_FILE PlateHeatExchanger.idf EPW_FILE USA_FL_Tampa.Intl.A
 ADD_SIMULATION_TEST(IDF_FILE Plenum.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE PlenumwithRetAirHeatGain.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE PondGroundHeatExchanger.idf EPW_FILE USA_AZ_Phoenix-Sky.Harbor.Intl.AP.722780_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE PTAC_ConnectedToDOAS.idf EPW_FILE USA_FL_Miami.Intl.AP.722020_TMY3.epw)
```

Comment:
Understood.  Note the old files are simpley "DOA" to . . ., not "DOAS".  Or we could rename the old ones, just seems the set should be consistent.